en/mulesoft/modules/ROOT/pages/ms_composer_salesforce_marketing_cloud_reference.adoc

```diff
@@ -2,6 +2,43 @@
 
 Salesforce Marketing Cloud is a marketing platform that supports capabilities such as real-time customer engagement, email and marketing automation, social media engagement, mobile (SMS) messaging and push notifications, and customer marketing analytics.
 
+
+== Create API Integration Package and Set Permissions
+
+To use the Salesforce Marketing Cloud Connector, you must first create an API Integration package and set permissions for email and SMS communication.
+
+To create a package and set the permissions:
+
+. In Composer, click the gear icon in the upper right corner. +
+The *Setup* page appears.
+
+. On the left navigation pane, under *Platform Tools*, click *Apps* > *Packaging* > *Installed Packages*. +
+The *Installed Packages* page appears.
+
+. In the upper-right corner of the *Installed Packages* page, click *New*. +
+The *Package Details* window appears.
+
+. In the *Package Details* window, complete the following fields:
+.. *Name*: Enter a unique name for the new package.
+.. *Description*: Enter a description of the package.
+
+. Click *Save*. +
+The *Summary* page appears displaying the details of the new package.
+
+. In the *Summary* page, click *Add Component*. +
+The *Add Component* window appears.
+
+. In the *Add Component* window, select *API Integration* > *Server-to-Server*. +
+The *Set Server-to-Server* section appears.
+
+. In the *Set Server-to-Server* section, under *Channels*, click the checkboxes next to the following:
```

Comment:
```suggestion
. In the *Set Server-to-Server* section, under *Channels*, select the checkboxes next to the following:
```
Click could be ambiguous for checkboxes since it does not specify if they should remain enabled/disabled.